deploy/mpi-operator.yaml

```diff
@@ -0,0 +1,194 @@
+# --------------------------------------------------
+# - Single configuration deployment YAML for MPI-Operator
+# - Include:
+#      CRD deployment
+#      Namespace deployment
+#      RBAC deployment
+#      MPI-operator container deployment set
+# --------------------------------------------------
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: mpijobs.kubeflow.org
+spec:
+  group: kubeflow.org
+  version: v1alpha2
```

Comment:
This is only the v1alpha2 CRD. Perhaps we should only combine the yaml files besides the CRDs so we don't need to keep two files with a lot of overlapped content, e.g RBAC, etc.

Reply:
CRD with v1alpha1 is supported by mpioperator/mpi-operator:latest or customer must use mpioperator/mpi-operator:v1alpha1 version?

Reply:
Sorry I somehow missed this question. No, v1alpha1 would not work with the latest images.

Reply:
So, we can leave only the v2 version on the master branch. But single yaml deployment file will make deployment easier.

Reply:
Do you know how to resolve LGTM label issue?

Reply:
cc @rongou to double check whether we need to make v1 version of it and whether broken existing links are fine.

Reply:
Yeah I think it's fine to ignore v1.

Reply:
Hi,
How will we continue? Do I need to do something?

Reply:
Thank you! Approving now.